data/nmstate/001-rbac

```diff
@@ -0,0 +1,64 @@
+{{ if .EnableSCC }}
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: nmstate-state-controller
```

Comment:
I think we can change it to `nmstate-controller`

Reply:
I think we should keep it nmstate-state-controller. we will introduce nmstate-policy-controller once it is ready

Reply:
any conclusion on what's preferred?

Reply:
Oh I did remember we have another component call policy so yes please leave it like that.